src/commands/es/list-index.ts

```diff
@@ -0,0 +1,60 @@
+import { flags } from '@oclif/command'
+import { Kommand } from '../../common'
+import { Client } from '@elastic/elasticsearch'
+
+export default class EsListIndex extends Kommand {
+  static description = 'List available ES indexes'
+
+  static flags = {
```

Comment:
Since this is a TS file, this should expose an interface for flag named `IEsInsertIndexFlag` ?

Reply:
Ditto

Reply:
Ditto